### ISSUE TITLE
Investigate appearance of additional "Holds" and "Availability" lines for digital materials

### ISSUE THREAD

**Issue by danfox01** [baltimorecounty/BCPL-assets#450]
OverDrive and cloudLibrary (ebooks and audiobooks) are now showing two lines of availability and holds, wheras BCPL reports there used to just be one line. Apparently there are two different values for digital materials -- one for BCPL and one for the entire Maryland system. For whatever reason, only BCPL's numbers were shown before and now both numbers are showing.

Also noteworthy... these numbers appear then quickly disappear in IE11. In Chrome and FF, they show the entire time.

Requires further investigation with internal developers as well as vendors.

**Comment by martypowell** [comment]
@danfox01 I believe we researched this last Friday and couldn't come up with anything. Next step to reach out to the vendor?

**Comment by danfox01** [comment]
@martypowell , yeah I've been meaning to talk to Karrie. Meanwhile, one additional quirk. In IE11, this issue behaves a little differently. Use the link above and note that the two additional lines show a loading spinner but then disappear. If you click on a record, the lines show again. This is still not what's supposed to happen, but it's different than what occurs in Chrome. Does this give you any more clues before I talk to the vendor?
